docs/build.md

```diff
@@ -282,9 +282,13 @@ The following compilation options are also available to tweak performance (yes,
 
 #### w64devkit
```

Comment:
I moved `w64devkit` above the example i just created.

I think its useful to have another example, I had most of this already installed.